webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -842,6 +842,21 @@ private static void fitAndShowText(PdfContentByte cb, String text, BaseFont bf,
         } while(true);
     }
     
+    private static void populateRect(PdfContentByte cb, Rectangle rect, ArrayList<String> list, ArrayList<Integer> alignList, BaseFont bf, int fontSize)  throws DocumentException {
+
```

Comment:
Can you add an assertion here that `list.size() == alignList.size()`?

Reply:
Ok =)